pkg/square/builder.go

```diff
@@ -339,6 +339,14 @@ func (b *Builder) GetWrappedPFB(txIndex int) (*coretypes.IndexWrapper, error) {
 	return b.pfbs[txIndex-len(b.txs)], nil
 }
 
+func (b *Builder) NumPFBs() int {
+	return len(b.pfbs)
+}
+
+func (b *Builder) NumTxs() int {
+	return len(b.txs)
+}
```

Comment:
[optional] I think it may help if we consistently referred to the non-blob Txs as something like `normalTxs` rather than just `txs`. This proposal is mostly b/c I think of it like: pfbTxs + normalTxs = txs and the way this is written `txs` is ambiguous if it refers to normalTxs or the set of all Txs

Reply:
Yeah I can adopt that nomenclature